controllers/gslb_controller_test.go

```diff
@@ -849,6 +849,78 @@ func TestCreatesDNSNSRecordsForExtDNS(t *testing.T) {
 	assert.Equal(t, wantEp, gotEp, "got:\n %s DNSEndpoint,\n\n want:\n %s", prettyGot, prettyWant)
 }
 
+func TestCreatesDNSNSRecordsForLoadBalancer(t *testing.T) {
+	// arrange
+	const dnsZone = "cloud.example.com"
+	const want = "extdns"
+	wantEp := []*externaldns.Endpoint{
+		{
+			DNSName:    dnsZone,
+			RecordTTL:  30,
+			RecordType: "NS",
+			Targets: externaldns.Targets{
+				"gslb-ns-eu-cloud.example.com",
+				"gslb-ns-us-cloud.example.com",
+				"gslb-ns-za-cloud.example.com",
+			},
+		},
+		{
+			DNSName:    "gslb-ns-eu-cloud.example.com",
+			RecordTTL:  30,
+			RecordType: "A",
+			Targets: externaldns.Targets{
+				defaultEdgeDNS1,
+			},
+		},
+	}
+	dnsEndpoint := &externaldns.DNSEndpoint{}
+	customConfig := predefinedConfig
+	customConfig.EdgeDNSServers = defaultEdgeDNSServers
+	customConfig.CoreDNSExposed = true
+	coreDNSService := &corev1.Service{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      defaultCoreDNSExtServiceName,
+			Namespace: predefinedConfig.K8gbNamespace,
+			Labels: map[string]string{
+				"app.kubernetes.io/name": "coredns",
+			},
+		},
+	}
+	serviceIPs := []corev1.LoadBalancerIngress{
+		{IP: "1.1.1.1"}, // rely on 1.1.1.1 response from Cloudflare
```

Comment:
@k0da , are we really checking resolution by CloudFlare here or are these just mock test data?
https://blog.cloudflare.com/cloudflare-outage-on-june-21-2022/

Reply:
This is just a test data

Reply:
@k0da if it's just test data, then the comment describing that it's not should be removed. Looks like a copy-paste from https://github.com/k8gb-io/k8gb/blob/master/controllers/gslb_controller_test.go#L818 where we actually relying on Cloudflare's FQDN resolution